examples/http-compression/tests/http_compression.bal

```diff
@@ -0,0 +1,29 @@
+import ballerina/test;
```

Comment:
Shall we add some comments to this example?

Reply:
This is the test file of example. It is not exposed externally